Submodule Poverty contains modified content
diff --git a/Poverty/CMakeLists.txt b/Poverty/CMakeLists.txt
index f2e61f3..8ba188a 100644
--- a/Poverty/CMakeLists.txt
+++ b/Poverty/CMakeLists.txt
@@ -31,6 +31,6 @@ set( SERVER_SHARED_LIB_TARGET ${EXECUTABLE}Object )
 add_library( PovertyPython SHARED
         ${PROJECT_BINARY_DIR}/Poverty/output/PovertyPythonModule.cpp
         )
-target_link_libraries(PovertyPython boost_python-mt ${SERVER_SHARED_LIB_TARGET})
+target_link_libraries(PovertyPython boost_python3 ${SERVER_SHARED_LIB_TARGET})
 set_target_properties(PovertyPython PROPERTIES PREFIX "" OUTPUT_NAME "Poverty") 
-add_dependencies(PovertyPython ${SERVER_SHARED_LIB_TARGET})
\ No newline at end of file
+add_dependencies(PovertyPython ${SERVER_SHARED_LIB_TARGET})
